example/storybook/src/components/Disclosure/Actionsheet/index.stories.mdx

```diff
@@ -609,9 +609,8 @@ function App(){
                     Confirm security code
                   </FormControl.Label.Text>
                 </FormControl.Label>
-                <Input w="$full">
-                 
-                  <Input.Input placeholder="CVC/CVV" />
+                <Input w="$full" mt={5}>
```

Comment:
Generally we don't take fullWidth in Input

Reply:
okay